jina/parsers/orchestrate/deployment.py

```diff
@@ -43,6 +43,14 @@ def mixin_base_deployment_parser(parser):
         'This Deployment will not be context managed by the Flow.',
     )
 
+    gp.add_argument(
+        '--grpc-metadata',
+        action=KVAppendAction,
+        metavar='KEY: VALUE',
+        nargs='*',
+        help='The metadata to be passed to the gRPC request. This argument only applies for external Deployments.',
```

Comment:
correct the message, it will apply to anything. One thing is the argument and the other is the feature it enables, which should be clarified in Documentatiok

Reply:
addressed